meetings/2025-04-09.md

```diff
@@ -0,0 +1,120 @@
+* Status: Agenda
+
+
+## Present
+* [Michiel de Jong](https://michielbdejong.com)
+* [Thhck](https://github.com/thhck)
+* Alain Bourgeois
+* [Matthias Evering](https://solidweb.me/testpro/)
+* Hadrian Zbarcea
+* [elf Pavlik](https://elf-pavlik.hackers4peace.net)
+* Jeff Zucker
+* [Erich Bremer](https://ebremer.com/profile#me)
+* [TallTed // Ted Thibodeau Jr](https://github.com/TallTed/) (he/him) ([OpenLink Software](https://www.openlinksw.com/))
+* Jesse Wright
+
+## Announcements
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/w3c-cg/solid/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+
+* elf Pavlik
+
+## Topics
+
+### Peaceful Progress
+(topic moved forward from [21 may 2025 agenda](https://hackmd.io/dtohRN8ZSVqBgRElSwEZlw))
+
+* MdJ: In the interest of our community's social harmony, let's focus our work as a CG on topics that don't overlap with topics that other groups within the Solid Community may have principled opinions about.
+  ... And it's not just about the social harmony. There is also a technical requirement. The power of Solid is that we're creating a _standard_ way for folks to run and use their personal online data stores. Creating a non-standard way to do that would be trivial, but creating a standard way requires convergence on many levels (client-server as well as client-client).
+  ... How can we achieve this convergence which is so essential for Solid? Would it for instance be better for the Solid Project as a whole if for instance Solid CG and Solid Lab Flanders would coordinate more? Or if Solid OS would coordinate more with SAI? Otherwise I fear we can continue to incubate many parallel experiments for a long time, without delivering on Solid's promise.; [JZ: and relative scope of CG compared to Practitioners]
```

Comment:
```suggestion
  ... How can we achieve this convergence which is so essential for Solid? Would it be better for the Solid Project as a whole if, for instance, Solid CG and Solid Lab Flanders would coordinate more? Or if Solid OS would coordinate more with SAI? Otherwise I fear we will continue to incubate many parallel experiments for a long time, without delivering on Solid's promise. [JZ: and relative scope of CG compared to Practitioners]
```